resume.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+
+    <title>My website | Resume</title>
+  </head>
+
+  <body>
+    <div class=".container">
+      <!-- Header section -->
+      <header>
+        <nav class="nav second-nav">
+          <input
+            type="checkbox"
+            name=""
+            id="nav-checkbox"
+            class="nav-checkbox"
+          />
+          <label for="nav-checkbox" class="nav-toggle">
+            <img
+              class="menu"
+              src="./img/burger-menu.svg"
+              width="100"
+              alt="burger menu image"
+            />
+            <svg class="close" viewBox="0 0 384 512" width="100" title="times">
+              <path
+                d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
+              />
+            </svg>
+          </label>
+          <ul class="nav-menu">
+            <li>
+              <a href="#"><div class="square"></div></a
+              ><strong>Maya Nelson</strong> <br /><span class="occupation"
+                >PROJECT MANAGER</span
+              >
+            </li>
+            <li><a href="index.html">ABOUT ME</a></li>
+            <li><a class="active" href="resume.html">RESUME</a></li>
+            <li><a href="projects.html">PROJECTS</a></li>
+            <li><a href="contact.html">CONTACT</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- Main section - Resume -->
+      <main>
+        <!-- Resume -->
+        <section class="resume bolder">
+          <div class="title primary-font">
+            <div class="square"></div>
+            <h1>Resume</h1>
+          </div>
+          <!-- Experience Section -->
+          <section class="experience">
+            <div class="exp-title">
+              <h2 class="primary-font">Experience</h2>
+              <a class="btn btn-primary" href="#">DOWNLOAD CV</a>
+            </div>
+            <div class="exp-articles">
+              <article class="article-1">
```

Comment:
some tips. if you wanna devide the section like the original website.
group two paragraphs (title-art1 and the below one) with `div` and add `display:flex;` to the div. flexbox is super useful 🤓

<img width="725" alt="스크린샷 2023-03-06 00 19 57" src="https://user-images.githubusercontent.com/63498427/222994610-8a5f5e64-8af3-463b-9dbf-70f7d0cc141c.png">